recipe/meta.yaml

```
@@ -6,38 +6,50 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 84b4c1d0e4168f7d30533b2c136f25a66f55e0e0d2c81b4748878c10165a31b8
+  url: https://github.com/Roche/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 987c102576af2cfc8656b81098eb52d1abf931953634985f16e0ed2032d8aa96
 
 build:
   number: 0
   skip: true  # [py<35]
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - cython                                 # [build_platform != target_platform]
-    - {{ compiler('c') }}  # [win]
-    - {{ compiler('c') }}  # [not win]
+    - {{ compiler('c') }}
   host:
     - python
     - pip
-    - cython
-    - zlib  # [not win]
-    - zlib  # [win]
-    - libiconv  # [win]
+    - setuptools
+    - wheel
+    - cython 3.0.6
+    - zlib 1.2.13
```

```suggestion
    - zlib {{ zlib }}
```
We should use the version from aggregate.